### ISSUE TITLE
Can we change marker genes used in scatter plots?

### ISSUE THREAD

**Issue by dioncst** [wguo-research/scCancer#11]
Can we set personalized marker genes showed in part 3.1 Markers expression profile ?

**Comment by wguo-research** [comment]
Yes, you can input your interested genes to "runScAnnotation" function by the argument "**show.features**". Using help(runScAnnotation) can get more detailed information.

**Comment by wguo-research** [comment]
The `cnv.ref.data` can be set as a count matrix of gene by cell. The row names are genes' symbol, and the column names are cells' label (such as "Normal1", "Normal2", etc). And `cnv.referAdjMat` can be set as `NULL` or a SNN matrix of reference data.
